openshift_tools/monitoring/zagg_sender.py

```diff
@@ -27,6 +27,14 @@
 import os
 import yaml
 
+# Add Hawkular sender backend (Optional)
```

Comment:
@simon3z 
I understood from @twiest that they want hawkular to be _off_ by default, and everything to work without change if hawkular client is not installed, or the backend is not activated in the conf.yaml file.

p.s.
If we move to using hawkular why we need to send data to zabbix ?